pom.xml

```diff
@@ -52,6 +52,7 @@
     <spring.version>4.1.4.RELEASE</spring.version>
     <fcrepo.version>4.4.0</fcrepo.version>
     <fcrepo-transform.version>4.4.0</fcrepo-transform.version>
+    <fcrepo-java-client.version>4.4.1-SNAPSHOT</fcrepo-java-client.version>
```

Comment:
+1 to the first option. The problem with the second option is that releases _must_ then be coordinated across projects, and it was decided (for good reasons) to allow independent projects to be versioned and released independently.